code/controllers/subsystem.dm

```diff
@@ -34,8 +34,7 @@
 	var/static/list/failure_strikes //How many times we suspect a subsystem type has crashed the MC, 3 strikes and you're out!
 
 //Do not override
-/datum/controller/subsystem/New()
-	return
+///datum/controller/subsystem/New()
```

Comment:
don't leave commented code out like this come on

Reply:
See the above line